code/variables.tf

```diff
@@ -299,6 +299,18 @@ variable "ad_key_vault_group_object_id" {
   description = "Active directory object id group that can access key vault."
 }
 
+variable "agid_spid_pem_name" {
+  type        = string
+  description = "Secret name with agid spid pem file content"
+  default     = null
+}
+
+variable "agid_spid_key_name" {
+  type        = string
+  description = "Secret name with agid spid key file content"
+  default     = null
+}
```

Comment:
Can we rename this in a more meaningful way? something like `agid_spid_public_cert` and `agid_spid_private_key` should suffice